src/ports/postgres/modules/deep_learning/test/unit_tests/test_madlib_keras.py_in

```diff
@@ -178,11 +178,19 @@ class MadlibKerasFitTestCase(unittest.TestCase):
     def test_split_and_strip(self):
         self.assertEqual(('a','b'), self.subject.split_and_strip(' a = b '))
 
-    def test_parse_fit_params(self):
+    def test_parse_and_validate_fit_params(self):
```

Comment:
We should also add negative test cases for `parse_and_validate_fit_params` and `parse_compile_params`. Maybe not now but in a future PR